platform-common/src/main/java/com/softicar/platform/common/core/locale/Locale.java

```diff
@@ -13,13 +14,17 @@ public class Locale implements ILocale {
 	private String decimalSeparator;
 	private String digitGroupSeparator;
 	private String dateFormat;
+	private String localizedDateFormat;
+	private String localizedTimeFormat;
```

Comment:
- why do we have `dateFormat` and `localizedDateFormat`?
- not sure why we need the `localized` prefix? everything in here is localized

Reply:
`dateFormat` contains format codes of `java.text.SimpleDateFormat` which will appear odd or even incomprehensible to ordinary mortals; particularly if they use the system in a non-English language:
- When `yyyy-MM-dd` is displayed to an English-speaking user, they will probably wonder why it's neither `yyyy-mm-dd` nor `YYYY-MM-DD`. It looks inconsistent, and like a mistake.
- When `dd.MM.yyyy` is displayed to a German-speaking user, capitalization might be the least of their issues because they'd expect something like `TT.MM.JJJJ`. They might even fail to comprehend the format because of this; so displaying it (which we do in those pseudo tooltips) would be kind of pointless.

That being said, quoting from the Javadoc of `ILocale`:
```
/**
 * Returns a human-readable date format that uses abbreviations which
 * correspond to the language returned by {@link #getLanguage()}.
 *
 * @return a localized, human-readable date format (never <i>null</i>)
 */
String getLocalizedDateFormat();

/**
 * Returns a human-readable time format that uses abbreviations which
 * correspond to the language returned by {@link #getLanguage()}.
 *
 * @return a localized, human-readable time format (never <i>null</i>)
 */
String getLocalizedTimeFormat();
```

I'm not entirely happy with the naming either... I used "informal" instead of "localized" before. Any thoughts on that?